tests/app/main/views/test_jobs.py

```diff
@@ -260,6 +261,7 @@ def test_should_show_page_for_one_job_with_flexible_data_retention(
     )
 
     assert page.find('span', {'id': 'time-left'}).text == 'Data available for 10 days'
+    assert "Cancel sending those letters" not in page
```

Comment:
This would be a bit more robust:
```suggestion
    assert "Cancel sending these letters" not in page
    assert page.select_one('.page-footer-delete-link') is None
```